dali/operators/reader/loader/video_loader.h

```diff
@@ -245,15 +251,33 @@ class VideoLoader : public Loader<GPUBackend, SequenceWrapper> {
         } else {
           auto frame_rate = av_inv_q(file.frame_base_);
           if (start >= 0) {
-            start_frame = static_cast<int>(std::ceil(start * av_q2d(frame_rate)));
+            if (file_list_include_preceding_frame_) {
```

Comment:
Can you make those if consistent?
Once it's `if (file_list_include_preceding_frame_)` on top level, the other time it's the nested one.

Reply:
Done